include/libvfio-user.h

```diff
@@ -240,6 +245,26 @@ typedef ssize_t (vfu_region_access_cb_t)(vfu_ctx_t *vfu_ctx, char *buf,
  *    and writes are an error
  *  - otherwise, the callback is expected to handle the access
  *
+ * Regions VFU_PCI_DEV_MIGR_REGION_IDX and VFU_GENERIC_DEV_MIGR_REG_IDX,
+ * corresponding to the migration region, enable live migration support for
+ * the device. The migration region must contain at the beginning the migration
+ * registers (struct vfio_device_migration_info defined in <linux/vfio.h>) and
+ * the remaining part of the region can be arbitrarily used by the device
+ * implementation. Therefore, the size of the migration region must be at least
+ * sizeof(struct vfio_device_migration_info). libvfio-user offers two ways for
```

Comment:
New para here? it's a bit hard to follow all in one

Reply:
ok